core/errors.go

```diff
@@ -108,3 +108,24 @@ var (
 	// ErrBlobTxCreate is returned if a blob transaction has no explicit to field.
 	ErrBlobTxCreate = errors.New("blob transaction of type create")
 )
+
+// TxIndexingError is an API error that indicates the transaction indexing is not
+// fully finished yet with JSON error code and a binary data blob.
+type TxIndexingError struct{}
```

Comment:
It's a geth customized error for serving RPC purposes. I don't think this definition should be mixed with other errors (e.g. consensus error)